staging/src/k8s.io/cloud-provider/controllers/node/node_controller_test.go

```diff
@@ -1294,10 +1298,14 @@ func Test_AddCloudNode(t *testing.T) {
 				recorder:                  eventBroadcaster.NewRecorder(scheme.Scheme, v1.EventSource{Component: "cloud-node-controller"}),
 				nodeStatusUpdateFrequency: 1 * time.Second,
 			}
+
+			cloudNodeController.nodeInformer.Informer().GetIndexer().Add(test.existingNode)
+			cloudNodeController.nodesSynced = alwaysReady
```

Comment:
Do we need to mock `nodesSynced` here? I think since we're not calling `Run` and only `SyncNode`, we don't need to set this.

Reply:
gotcha!